specs/src/specs/namespace.md

```diff
@@ -34,14 +48,27 @@ A namespace with version `0` must contain an id with a prefix of 18 leading `0`
 0x1111111111111111111111111111111111111111111111111111111111 // invalid because it does not have version 0
 ```
 
-A new namespace version MUST be introduced if the namespace format changes in a backwards incompatible way (i.e. the number of leading `0` bytes in the id prefix is reduced).
+Any change in the number of leading `0` bytes in the id of a namespace with version `0` is considered a backwards incompatible change and MUST be introduced as a new namespace version.
 
 ### ID
 
-The namespace ID is a 28 byte identifier that uniquely identifies a namespace. The ID is encoded as a byte slice of length 28.
+The namespace ID is a 28 byte identifier that uniquely identifies a namespace.
+The ID is encoded as a byte slice of length 28.
+<!-- It may be useful to indicate the endianness of the encoding) -->
 
 ## Reserved Namespaces
 
+Celestia reserves certain namespaces with specific meanings.
+Applications SHOULD refrain from using these reserved namespaces for their blob data. <!-- the implication of this is still under investigation, but it is prudent to advise app developers not using this namespace -->
+The rationale behind this is that these namespaces dictate the positioning of transactions and blobs data within the Celestia [data square](./data_square_layout.md).
+Therefore, by refraining from using these reserved namespaces, applications can guarantee the desired arrangement of their blob data.
```

Comment:
[question] I don't understand this line. May it be rephrased?

Reply:
I decided to remove this part until the consequences of using reserved namespaces (https://github.com/celestiaorg/celestia-app/issues/2127) become more clear.